docs/_includes/ds6/listbox.html

```diff
@@ -0,0 +1,185 @@
+
+<div id="listbox">
+    <h2>@ebay/skin/listbox</h2>
+    <p>A listbox follows the principal of <a href="https://en.wikipedia.org/wiki/Progressive_disclosure">progressive disclosure</a>, it keeps secondary commands hidden from view until needed.</p>
+    <p>A listbox may be used as a form control. It is designed as a custom-styled component (much like a <a href="#menu">menu</a>, but different) on top of of a hidden input.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="listbox">
+                <button class="listbox__button expand-btn" aria-haspopup="listbox1" type="button">
+                    <span class="expand-btn__cell">
+                        <span>Listbox</span>
+                        <span class="expand-btn__icon"></span>
+                    </span>
+                </button>
+                <div id="listbox1" class="listbox__options" role="listbox">
+                    <div class="listbox__option" role="option">
+                        <span>Item 1</span>
+                    </div>
+                    <div class="listbox__option" role="option">
+                        <span>Item 2</span>
+                    </div>
+                    <div class="listbox__option" role="option">
+                        <span>Item 3</span>
+                    </div>
+                </div>
+                <select name="nativeOptions" hidden>
+                    <option value="item1">Item 1</option>
+                    <option value="item2">Item 2</option>
+                    <option value="item3">Item 3</option>
+                </select>
+            </span>
+        </div>
+        {% highlight html %}
+<span class="listbox">
+    <button class="listbox__button expand-btn" aria-haspopup="listbox1" type="button">
+        <span class="expand-btn__cell">
+            <span>Listbox</span>
+            <span class="expand-btn__icon"></span>
+        </span>
+    </button>
+    <div id="listbox1" class="listbox__options" role="listbox">
+        <div class="listbox__option" role="option">
+            <span>Item 1</span>
+        </div>
+        <div class="listbox__option" role="option">
+            <span>Item 2</span>
+        </div>
+        <div class="listbox__option" role="option">
+            <span>Item 3</span>
+        </div>
+    </div>
+    <select name="nativeOptions" hidden>
+        <option value="item1">Item 1</option>
+        <option value="item2">Item 2</option>
+        <option value="item3">Item 3</option>
+    </select>
+</span>
+        {% endhighlight %}
+    </div>
+
+    <h3>Selected Listbox Option</h3>
```

Comment:
I'm not seeing the icon for selected state:

![screen shot 2018-01-03 at 10 11 32 am](https://user-images.githubusercontent.com/38065/34533314-9604df10-f06e-11e7-868c-91aa78445bf1.png)

Reply:
Safari?

Reply:
Safari, firefox and chrome for me.

Reply:
Maybe related to selected/aria-selected issue?

Reply:
Yeah, got this fixed.

Reply:
Fixed.